src/Microsoft.AspNet.Identity/RoleManager.cs

```diff
@@ -60,6 +66,11 @@ public class RoleManager<TRole> : IDisposable where TRole : class
         protected IRoleStore<TRole> Store { get; private set; }
 
         /// <summary>
+        ///     Used to create scopes and messages for logging
+        /// </summary>
+        protected internal virtual IdentityLogger IdentityLogger { get; set; }
```

Comment:
Besides that, I am not sure we need the IdentityLogger class (I just wrote a comment for it), so this could end up being a regular ILogger.